README.md

```diff
@@ -126,6 +126,7 @@ The repository has Visual Studio project files for all examples and individually
     - To build in Release mode pass the `/p:Configuration=Release` option to MSBuild.
     - The exutables will be created in a subfolder named "Debug" or "Release" inside the project folder.
 - The HIP specific project settings like the GPU architectures targeted can be set on the `General [AMD HIP C++]` tab of project properties.
+- The top level solution files come in two flavors: `ROCm-Examples-VS<Visual Studio Verson>.sln` and `ROCm-Examples-Portable-VS<Visual Studio Version>.sln`. The former contains all examples, while the latter contains the examples that support both ROCm and CUDA.
```

Comment:
\+ @saadrahim,  @hchevanne

I don't recall that we discussed the separation of the solution. Whose decision was it?

Btw, the separation doesn't prevent the users from trying the HIP_nvcc platform toolset.

Reply:
I am against removing `<ProjectExcludedFromBuild>true</ProjectExcludedFromBuild>` from the projects which are not yet being correctly built on the `HIP_nvcc` path. The exclusion tag should be retained in the corresponding vcxproj files even with the separation of the solutions. I believe we still need an explicit warning on trying to build them. Otherwise, the users will report compilation errors, which we continue to overcome internally.

Reply:
I have restored the `ProjectExcludedFromBuild` tags. The portable solution file contained some examples it shouldn't have, this is now fixed also. 

From what I can tell (was not involved in the discussion), the portable solution file was introduced based on an internal ticket that required selecting one solution for all `HIP_nvcc`-capable examples.